src/grocker/resources/docker/compiler-image/compile.py

```diff
@@ -70,7 +70,11 @@ def setup_venv(python):
     """Setup venv and return python interpreter."""
     info('Setup venv using %s...', python)
     venv = tempfile.mkdtemp(suffix='.venv')
-    subprocess.check_call([python, '-m', 'virtualenv', '-p', python, venv])
+    try:
+        # python 3
+        subprocess.check_call([python, '-m', 'venv', '-p', python, venv])
```

Comment:
Le `-p python` n'est plus utile, si ? D'ailleurs, il ne l'a peut-être jamais été...

Reply:
indeed I remove it